client/tests/webdriver/baseSpecs/userLogin.spec.js

```diff
@@ -7,7 +7,21 @@ describe("Anet home page", () => {
     await (await Home.getBannerDropdown()).click()
     await (await Home.getLogoutButton()).waitForDisplayed()
     await (await Home.getLogoutButton()).click()
-    await Home.waitForLoginForm()
+    const title = await browser.getTitle()
+    await expect(title).to.equal("ANET Logout")
+    await Home.logout()
+  })
+  it("should be able to login from logout page", async() => {
```

Comment:
If you change the test title to "should be able to logout using the dropdown in header and login from logout page" then you can completely remove the `it("should be able to logout using the dropdown in header", …` test directly above, since it's totally covered by this test.